src/prefect/infrastructure/kubernetes.py

```diff
@@ -117,8 +118,8 @@ class KubernetesJob(Infrastructure):
 
     # controls the behavior of execution
     job_watch_timeout_seconds: int = Field(
-        default=5,
-        description="Number of seconds to watch for job creation before timing out.",
+        default=None,
+        description="Number of seconds to watch job execution before timing out.",
```

Comment:
Let's explain what happens if it's `None` and what the default behavior is in this description.

Reply:
Does this get reset each time the job emits an event?

Reply:
yes, `timeout_seconds` (if not `None`) should be enforced while waiting for each event to be emitted from the loop in [stream](https://github.com/kubernetes-client/python/blob/5a96bbcbe21a552cc1f9cda13e0522fafb0dbac8/kubernetes/base/watch/watch.py#L163)